yarn.lock

```diff
@@ -5747,7 +5747,7 @@ eslint@^7.23.0:
     text-table "^0.2.0"
     v8-compile-cache "^2.0.3"
 
-esm@agoric-labs/esm#Agoric-built:
+esm@^3.2.25, esm@agoric-labs/esm#Agoric-built:
```

Comment:
I think this should only be `esm@agoric-labs/esm#Agoric-built`

Reply:
done